web/book/src/reference/spec/type-system.md

```diff
@@ -223,6 +223,62 @@ of entries, which can be of different types. Thus, it is a tuple.
 select [1.4, false, "foo"]
 ```
 
+### Normalization
+
+Say we have a type expression E, composed of possibly many operators. To
```

Comment:
I think it might be helpful to put the PR description here, giving the times where we do the `-` operation. I'm still thinking through it. It sounds like:
- The initial examples concern primitive types
- Then the subsequent examples also are about the type of tuples, where "type" also includes the containing columns

Is that the right way to think about them?